src/test/java/io/jenkins/plugins/grading/ScoreTest.java

```diff
@@ -55,4 +57,36 @@ void shouldUpdateCoverage() {
 
         assertThat(score.getAchieved()).isEqualTo(98);
     }
+
+    @Test
+    void shouldUpdateTests() {
+        TestConfiguration testConfiguration = new TestConfigurationBuilder()
+                .setMaxScore(100)
+                .setFailureImpact(-3)
+                .build();
+
+        TestScore testScore = mock(TestScore.class);
+        when(testScore.getTotalImpact()).thenReturn(-3);
+
+        Score score = new Score();
+        score.addTestsTotal(testConfiguration, testScore);
+
+        assertThat(score.getAchieved()).isEqualTo(97);
```

Comment:
Habe ich bei meinem Beispiel auch vergessen: AssertJ Custom Assertions nutzen:
```suggestion
        assertThat(score).hasAchieved(97);
        assertThat(score).hasTotal(100);
```

Reply:
Vielleicht auch noch die anderen dazugehörigen Getter testen (habe ich bei meinem Beispiel auch vergessen).